pkg/api/v1/status/condition.go

```diff
@@ -82,6 +83,11 @@ const (
 	StreamConnectionReadyType ConditionType = "StreamConnectionReady"
 )
 
+const (
+	SearchIndexesNamesAreNotUnique = "SearchIndexesNamesAreNotUnique"
+	SearchIndexesSomeNotReady      = "SearchIndexesSomeNotReady"
```

Comment:
I am having doubts this is good UX 🤔 a negating status condition is a mental burden, i.e. `SearchIndexesSomeNotReady=False` indicates that everything is ok.

Why not simply `SearchIndexesReady`? The plural indicates that this refers to more than one and `SearchIndexesReady=True` is a clear indicator that everything is ok.

Addtionally, during e2e testing I found that there are double entries:
```
  status:
    conditions:
    - lastTransitionTime: "2024-05-03T05:43:35Z"
      message: not all indexes are in READY state
      reason: SearchIndexesSomeNotReady
      status: "False"
      type: Ready
    - lastTransitionTime: "2024-05-03T05:32:23Z"
      status: "True"
      type: ResourceVersionIsValid
    - lastTransitionTime: "2024-05-03T05:32:23Z"
      status: "True"
      type: ValidationSucceeded
    - lastTransitionTime: "2024-05-03T05:43:37Z"
      message: not all indexes are in READY state
      reason: SearchIndexesSomeNotReady
      status: "False"
      type: DeploymentReady
    - lastTransitionTime: "2024-05-03T05:43:37Z"
      message: IN_PROGRESS
      reason: InProgress
      status: "False"
      type: AtlasSearchIndexes

```